src/js/components/covid19/recipient/SpendingByRecipient.jsx

```diff
@@ -0,0 +1,102 @@
+/**
+ * SpendingByRecipient.jsx
+ * Created by Lizzie Salita 7/8/20
+ */
+
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { awardTypeTabs } from 'dataMapping/covid19/covid19';
+import { awardTypeGroups } from 'dataMapping/search/awardType';
+import { fetchRecipientCount } from 'helpers/disasterHelper';
+import SummaryInsightsContainer from 'containers/covid19/SummaryInsightsContainer';
+import SpendingByRecipientContainer from 'containers/covid19/recipient/SpendingByRecipientContainer';
+import AwardFilterButtons from './AwardFilterButtons';
+
+const overviewData = [
+    {
+        type: 'resultsCount',
+        label: 'Number of Recipients'
+    },
+    {
+        type: 'awardObligations',
+        label: 'Award Obligations',
+        dollarAmount: true
+    },
+    {
+        type: 'awardOutlays',
+        label: 'Award Outlays',
+        dollarAmount: true
+    },
+    {
+        type: 'numberOfAwards',
+        label: 'Number of Awards'
+    }
+];
+
+const SpendingByRecipient = () => {
+    const [activeTab, setActiveTab] = useState(awardTypeTabs[0].internal);
+    const defCodes = useSelector((state) => state.covid19.defCodes);
+
+    const [tabCounts, setTabCounts] = useState({
+        all: null,
+        grants: null,
+        direct_payments: null,
+        loans: null,
+        other: null,
+        contracts: null,
+        idvs: null
+    });
+
+    const changeActiveTab = (tab) => {
+        const selectedTab = awardTypeTabs.filter((item) => item.internal === tab)[0].internal;
```

Comment:
could use `.find` here and it would do the same thing w.o needing to append the `[0]`